src/store/dtree.ts

```diff
@@ -163,6 +170,30 @@ class DtreeStore {
     return getQueryBuilder(toJS(statList))
   }
 
+  get attributeStatusToChange(): TPropertyStatus | undefined {
```

Comment:
This code (attributeStatusToChange, currentStepGroups, currentStepGroupToChange) moved to modal-edit.store.ts. Please merge changes from `develop`